File: 283. Move Zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        lenn = len(nums)
        while 0 in nums:
            nums.remove(0)
        while len(nums) != lenn:
            nums.append(0)
        return 

File: 283. Move Zeroes.swift
class Solution {
    func moveZeroes(_ nums: inout [Int]) {
        var j = 0
        var cnt = 0
        for num in nums {
            if num != 0 {
                nums[j] = num
                j += 1
            } else {
                cnt += 1
            }
        }
        if (cnt > 0){
           for i in 1 ... cnt{
                nums[nums.count - i] = 0
            } 
        }
        return
    }
}
